include/fmt/ostream.h

```diff
@@ -147,6 +147,19 @@ struct fallback_formatter<T, Char, enable_if_t<is_streamable<T, Char>::value>>
     return std::copy(buffer.begin(), buffer.end(), ctx.out());
   }
 };
+
+template <typename It, typename Sentinel, typename Char>
+struct fallback_formatter<arg_join<It, Sentinel, Char>, Char>
```

Comment:
`fallback_formatter` is not an extension point so it shouldn't be specialize for `arg_join`. Instead make `formatter` specialization enabled if there is a fallback formatter for the element type as it is done for ranges.

Reply:
Ok hope it is better now.